Return options (#14)
ton-connect/docs: requests-responses.md

```tsx
type ConnectRequest = {
  manifestUrl: string;

>   returnStrategy: 'default' | 'disabled' | '<my-return-url>';

Reviewer: ```suggestion
  return: 'default' | 'disabled' | '<my-return-url>';
```

Author: Probably 'alwaysBack' will be clearer than 'back', what do you think?

Reviewer: What are the exact semantics of this? “Return where the system called from”? Then something like “system-back” would be more precise.

Not sure “always” has relevance here — this option is meant for a single interaction, it is not “sticky”.

}
```

ConnectRequest description:
- manifestUrl: link to the app's tonconnect-manifest.json

> - return: return strategy for deeplinks when user signs/declines the request. 'back' means return to the app which initialized deeplink jump (e.g. browser, native app, ...), 'none' means no jumps after user action. If a specific url is passed, wallet will open this url deeplink after user's action. Note, that you shouldn't pass your app url if it is a webpage. This option should be used for native apps to prevent 'back' strategy OS bugs.       

Reviewer: ```suggestion
- return: return strategy for deeplinks when user signs/declines the request. 
  * 'back' means return to the app which initialized deeplink jump (e.g. browser, native app, ...), 
  * 'none' means no jumps after user action;
  * a URL: wallet will open this URL after completing the user's action. Note, that you shouldn't pass your app's URL if it is a webpage. This option should be used for native apps to work around possible OS-specific issues with `'back'` option.
```

interface AppRequest {
	method: string;
	params: string[];

>     return: 'back' | 'none' | '<my-return-url>';

Reviewer: ```suggestion
	return: 'back' | 'none' | '<my-return-url>';
```

	id: string;
}
```
Where 
- method: name of the operation ('sendTransaction', 'singMessage', ...)
- params: array of the operation specific parameters

> - return: return strategy for deeplinks when user signs/declines the request. 'back' means return to the app which initialized deeplink jump (e.g. browser, native app, ...), 'none' means no jumps after user action. If a specific url is passed, wallet will open this url deeplink after user's action. Note, that you shouldn't pass your app url if it is a webpage. This option should be used for native apps to prevent 'back' strategy OS bugs.

Reviewer: Please format the list of options nicely, as mentioned above. Maybe even have a separate section for "Return options" and cross-ref it from both points in the spec.

Author: done
